test/testffmpeg.c

```diff
@@ -445,7 +445,7 @@ static AVCodecContext *OpenVideoStream(AVFormatContext *ic, int stream, const AV
             }
         } else
 #endif
-        if (vulkan_context && config->device_type == AV_HWDEVICE_TYPE_VULKAN) {
+            if (vulkan_context && config->device_type == AV_HWDEVICE_TYPE_VULKAN) {
```

Comment:
```suggestion
        if (vulkan_context && config->device_type == AV_HWDEVICE_TYPE_VULKAN) {
```

Reply:
Clangformat did that. If I fix it, it'll get put back again. Not sure what the right solution is..

Reply:
I've also no idea.
Usually you surround it with `/* clang-format off */` and `/* clang-format on */` but it feels overkill for this oneliner.

Reply:
Usually we don't run clang format because of things like this. :)
Go ahead and undo that change, thanks!